docs/introduction/quick-start.md

```diff
@@ -0,0 +1,300 @@
+# Quick Start
+
+This guide will help you quickly get started with **UniCore**. You will:
+
+1. Deploy a UniCore instance locally using Docker Compose.
+2. Issue a Credential to a user.
+3. Verify that Credential.
+
+### Prerequisites
+* [Docker](https://docs.docker.com/engine/install/)
```

Comment:
When I let Prettier format that file, I get a lot of extra blank lines added. Are you using any formatter for Markdown files yet? If not, would you like to use the Prettier plugin as well? Otherwise my IDE would automatically apply Prettier formatting when I touch this file next time.

Reply:
fixed